app/src/main/java/net/squanchy/venue/VenueInfoPageView.kt

```diff
@@ -0,0 +1,108 @@
+package net.squanchy.venue
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.os.Build
+import android.support.design.widget.CoordinatorLayout
+import android.support.v7.widget.Toolbar
+import android.text.Html
+import android.text.Spanned
+import android.util.AttributeSet
+import android.widget.ImageView
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import kotlinx.android.synthetic.main.merge_venue_info_layout.view.venueAddress
+import kotlinx.android.synthetic.main.merge_venue_info_layout.view.venueDescription
+import kotlinx.android.synthetic.main.merge_venue_info_layout.view.venueMap
+import kotlinx.android.synthetic.main.merge_venue_info_layout.view.venueName
+import net.squanchy.R
+import net.squanchy.home.Loadable
+import net.squanchy.imageloader.ImageLoader
+import net.squanchy.imageloader.ImageLoaderInjector
+import net.squanchy.navigation.Navigator
+import net.squanchy.support.ContextUnwrapper.unwrapToActivityContext
+import net.squanchy.venue.domain.view.Venue
+
+class VenueInfoPageView : CoordinatorLayout, Loadable {
+
+    constructor(context: Context, attrs: AttributeSet) : this(context, attrs, 0)
```

Comment:
We can probably remove this constructor and just annotate the other one with `@JvmOverloads`

Reply:
177667447d606c3fe4097de2e9b35b4df3d13ee0